<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>697</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>211</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titreJeu">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SOKOBAN</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>781</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QLabel" name="labNiv7">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="bNiv8">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Niveau 8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labNiv1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="bNiv6">
       <property name="text">
        <string>Niveau 6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bNiv7">
       <property name="text">
        <string>Niveau 7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="labNiv4">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="labNiv5">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="bNiv10">
       <property name="text">
        <string>Niveau 10</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="bNiv4">
       <property name="text">
        <string>Niveau 4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="labNiv8">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labNiv2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="labNiv3">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="bNiv9">
       <property name="text">
        <string>Niveau 9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="bNiv5">
       <property name="text">
        <string>Niveau 5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="labNiv9">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="bNiv3">
       <property name="text">
        <string>Niveau 3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labNiv6">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="bNiv1">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Niveau 1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="labNiv10">
       <property name="font">
        <font>
         <family>DejaVu Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="bNiv2">
       <property name="text">
        <string>Niveau 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bConfig">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Configurer la partie</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>121</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QPushButton" name="bNiveau">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Niveau 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="labNiveau">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bSuivant">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Suivant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bPrecedent">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Précédent</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigurer">
    <property name="title">
     <string>Configurer</string>
    </property>
    <addaction name="actionPartie"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionR_gle_du_jeu"/>
   </widget>
   <addaction name="menuConfigurer"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPartie">
   <property name="text">
    <string>Partie</string>
   </property>
  </action>
  <action name="actionR_gle_du_jeu">
   <property name="text">
    <string>Règle du jeu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
